docs/src/IceNucleation.md

```diff
@@ -208,3 +211,31 @@ Multiple sulphuric acid concentrations, ``x``,
     The current parameterization in CloudMicrophysics.jl is not valid for \Delta a_w
     values that are obtained from pure water droplets. Though CliMA lines look far
     from the Spichtinger 2023 line, the lines seem to move closer as x approaches 0.
+
+
+### INP Concentration Frequency
+
+With this parametrization, the concentration of ice nucleating particles (INPs) is found by randomly drawing from a relative frequency distribution, which depends on the temperature. It is based on [Frostenberg2023](@cite) and represents the randomness of INP concentrations. The scheme does not require information about aerosol. The relative frequency distribution ``D`` of INP concentration is derived from measurements, for marine
+data sets (as described in [Frostenberg2023](@cite)). It is a lognormal distribution, described by
+
+```math
+\begin{equation}
+  D(\mu, \sigma^2) = \frac{1}{\sqrt{2 \pi} \sigma} exp \left(- \frac{[ln(a \cdot INPC) - \mu(T)]^2}{2 \sigma^2} \right),
+\end{equation}
+```
+where ``T`` is the temperature in degrees Celsius, ``INPC`` is the INP concentration in m``^{-3}`` and the temperature-dependent mean ``\mu(T)`` is given by
+```math
+\begin{equation}
+  \mu(T) = ln(-(b \cdot T)^9 \times 10^{-9})
+\end{equation}
+```
+ ``\sigma^2`` is the variance, ``a`` and ``b`` are coefficients. The parameters defined in [Frostenberg2023](@cite) for marine data sets are ``\sigma^2=1.37``, ``a=1 m^3``, ``b=1 [/ \degree C]``.
+
```

Comment:
The degree symbol doesn't render properly on the preview. Maybe deg will work. If not, I'm fine to just have `C`

Reply:
I'm not sure how to see the preview, but I hope it'll work now

Reply:
No, it still doesn't render properly. Lets just do `1/C` and not spend time on this. 

Could I ask you to change the temperature in the plots to be in Kelvin?